src/overload_connectivity.jl

```diff
@@ -1,3 +1,5 @@
+# TODO: support Duals
```

Comment:
Actually for connectivity they are not needed, since by definition we consider it to be value-independent?

Reply:
It's necessary for control flow and comparisons. 
Could be useful for [data-flow analysis](https://en.wikipedia.org/wiki/Data-flow_analysis), but isn't really relevant to our goals.

Reply:
Duals on `ConnectivityTracer` have been added.